rclrs/src/clock.rs

```diff
@@ -45,19 +46,37 @@ impl Clock {
     }
 
     /// Creates a new Clock with `ClockType::SteadyTime`
+    /// to update it
+    ///
+    /// # Returns
+    ///
+    /// A tuple of Clock and ClockSource
     pub fn steady() -> Self {
         Self::make(ClockType::SteadyTime)
     }
 
     /// Creates a new Clock with `ClockType::RosTime` and a matching `ClockSource` that can be used
     /// to update it
+    ///
+    /// # Returns
```

Comment:
There really is no need for the `# Returns` section. Rustdoc already does a great job showcasing the function signature which will include the return type (with a link to it!).

Sometimes the function signature + name is sufficient documentation and anything we add is just noise.